program/styles/index.css

```diff
@@ -0,0 +1,289 @@
+@import url("./header.css");
+@import url("./select-program.css");
+@import url("./contacts.css");
+@import url("./comments.css");
+@import url("./additionally.css");
+@import url("./map.css");
+@import url("./footer.css");
+
+@font-face {
+  font-family: "Oswald";
+  font-style: normal;
+  font-weight: 400;
+  src: url(../fonts/Oswald/Oswald-Regular.woff2) format("woff2"),
+    url(../fonts/Oswald/Oswald-Regular.woff) format("woff");
+  font-display: swap;
+}
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+  font-weight: 400;
+  background-color: #ffffff;
+  font-family: "Oswald", "Open Sans", sans-serif;
+  text-transform: uppercase;
+  color: #111111;
+  z-index: 100;
+}
+
+.visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  border: 0;
+  padding: 0;
+  white-space: nowrap;
+  clip-path: inset(100%);
+  clip: rect(0 0 0 0);
+  overflow: hidden;
+}
+
+h1,
+h2,
+h3 {
+  margin: 0;
+}
+
+img,
+svg {
+  display: block;
+  max-width: 100%;
+  height: auto;
+}
+
+ul,
+ol {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+p {
+  margin: 0;
+  font: inherit;
+}
+
+b {
+  display: block;
+  font: inherit;
+}
+
+i {
+  font-style: normal;
+}
+
+a,
+button {
+  transition: all 0.1s ease-in;
+  text-decoration: none;
+  text-transform: uppercase;
+  outline: none;
+}
+
+a[href^="tel"] {
+  color: inherit;
+}
+
+button {
+  margin: 0;
+  padding: 0;
+  font: inherit;
+  color: inherit;
+  text-decoration: none;
+  background-color: rgba(0, 0, 0, 0);
+  box-shadow: none;
+  text-shadow: none;
+  border: 0;
+}
+
+button:hover {
+  cursor: pointer;
+}
+
+* {
+  -webkit-text-size-adjust: none;
+  box-sizing: border-box;
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 1220px;
+}
+
+.main__wrapper {
+  margin: 0 auto;
+  max-width: 1440px;
+}
+
+.description {
+  font-size: 16px;
+  line-height: 24px;
+  color: #444444;
+  font-family: "Arial", sans-serif;
+  text-transform: none;
+}
+
+.title {
+  font-size: 60px;
+  text-transform: none;
+}
+
+.subtitle {
+  font-size: 36px;
+  text-transform: none;
+  color: #111111;
+  line-height: 1;
+  text-align: left;
+  position: relative;
+  z-index: 300;
+}
+
+.subtitle__line {
+  width: 100vw;
+  position: relative;
+  left: 50%;
+  right: 50%;
+  margin-left: -50vw;
+  margin-right: -50vw;
+  border-bottom: 3px solid #68b738;
+  z-index: 100;
+  bottom: 14px;
+}
+
+.contacts__subtitle::before {
+  content: "";
+  position: absolute;
+  top: 22px;
+  left: -33px;
+  width: 585px;
+  height: 6px;
+  background-color: #ffffff;
+  z-index: -100;
+}
+
+.comments__subtitle::before {
+  content: "";
+  position: absolute;
+  top: 21px;
+  left: -33px;
+  width: 255px;
+  height: 6px;
+  background-color: #ffffff;
+  z-index: -100;
+}
+
+.additionally__subtitle::before {
+  content: "";
+  position: absolute;
+  top: 22px;
+  left: -33px;
+  width: 280px;
+  height: 6px;
+  background-color: #ffffff;
+  z-index: -100;
+}
+
+fieldset {
```

Comment:
If you specify styles for a tag, it should be in a reset section, not where you describe blocks and elements with CSS classes. It's a common thing which will affect all fieldsets across the website. So please put it there.